.github/workflows/linux.yaml

```diff
@@ -38,12 +38,10 @@ jobs:
         run: bundle install --jobs 4 --retry 3
       - name: Pre-test setup
         run: |
-          sudo curl -L https://github.com/docker/compose/releases/download/1.23.0/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose
```

Comment:
Learned that docker and docker-compose are already installed in the GH action virtual envs.